dataduct/steps/scripts/pipeline_dependency_check.py

```diff
@@ -100,8 +111,11 @@ def main():
     parser = argparse.ArgumentParser()
     parser.add_argument(
         '--dependencies', type=str, nargs='+', default=None)
+    parser.add_argument('--pipeline_name', dest='pipeline_name')
     parser.add_argument('--refresh_rate', dest='refresh_rate', default='900')
     parser.add_argument('--start_date', dest='start_date')
+    parser.add_argument('--ignore_dependencies', dest='ignore_dependencies', default='0')
```

Comment:
I think it would be more useful to be able to turn this off on a pipeline-by-pipeline basis, instead of providing one flag for all dependencies.

@sb2nov, thoughts?